model/src/lib.rs

```diff
@@ -529,6 +530,55 @@ impl str::FromStr for Identity {
 
 impl_sqlx_type_display_from_str!(Identity);
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Libp2pIdentity(PeerId);
+
+impl Libp2pIdentity {
+    pub fn peer_id(&self) -> PeerId {
+        self.0
+    }
+
+    // TODO: Change this to be const
+    /// Placeholder used for db entries prior to using libp2p
+    pub fn placeholder() -> Libp2pIdentity {
+        // peer id for key filled up with 0s
+        Self(
+            "12D3KooWE1krEN7utXUFhLmLXLWPSuah5WQ7ChzUFetkMS7TRXsm"
```

Comment:
🔧 Would be good if you could tie this value to the one you use in the test in `seed.rs`.

Reply:
I know, but it's hard because we moved the code to different `crates`. I don't really want to expose the `PlaceholderSeed` - I just added the test to showcase that the placeholder is derived deterministically. Could also remove it. The test does not prove anything.